lib/lang/latin/latin-language-dataset.js

```diff
@@ -368,8 +384,29 @@ export default class LatinLanguageDataset extends LanguageDataset {
     return this
   }
 
+  getVerbsIrregularLemma (verbForm) {
+    let res = this.pos.get(Constants.POFS_VERB).types.get(Form).items.map(item => { return { val: item.value, lemma: item.features[Feature.types.hdwd].value, fullForm: item.features[Feature.types.fullForm].value } })
+    let res1 = {}
+    res.forEach(function (item) {
+      res1[item.val] = item.lemma
+      res1[item.fullForm] = item.lemma
+    })
+
+    return res1[verbForm] ? res1[verbForm] : null
+  }
+
+  checkIrregularVerb (inflection) {
+    if (inflection.hasFeatureValue(Feature.types.part, Constants.POFS_VERB) && this.getVerbsIrregularLemma(inflection[Feature.types.fullForm])) {
```

Comment:
Can we simplify it to `return (inflection.hasFeatureValue(Feature.types.part, Constants.POFS_VERB) && this.getVerbsIrregularLemma(inflection[Feature.types.fullForm]))`?